float64[6] l_arm_p
float64[6] r_arm_p

float64[6] l_arm_v
float64[6] r_arm_v

float64[5] l_arm_acc
float64[5] r_arm_acc

float64[5] l_arm_dec
float64[5] r_arm_dec